receiver/apachereceiver/README.md

```diff
@@ -16,7 +16,7 @@ This receiver fetches stats from a Apache Web Server instance using the `server-
 
 ## Prerequisites
 
-This receiver supports Apache Web Server version 2.4+
+This receiver supports Apache Web Server version 2.4.13+.
```

Comment:
Raising the question again from https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39758#issuecomment-2841823663.

Is this ensured or required for all the documented metrics or supportability is different per metric? 

From the [description](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39758) that's not clear to me.

Reply:
2.4.13 is over 10 years old from what I can find, is giving more granularity important?

Reply:
In that case, probably not. My concern is more on why to select this version over another and what will happen in the future if new metrics are added that are only supported for x.y.z+ version.

Reply:
Fair!